pkgs/development/python-modules/nestedtext/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi
+, inform
+}:
+
+buildPythonPackage rec {
+  pname = "nestedtext";
+  version = "1.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "18q7zdzpx2ixka69hgmbw072bw7q5m4wyfplbznjmk8zk3vr3spd";
+  };
+
+  propagatedBuildInputs = [ inform ];
+
+  meta = with lib; {
+    description = "A human friendly data format";
+    homepage = "https://nestedtext.org";
```

Comment:
Can I make a suggestion: What about including the first paragraph of documentation as a `longDescription`? It can be useful for a more curious person.

```
NestedText is a file format for holding data that is to be entered, edited, or viewed by people. It allows data to be organized into a nested collection of dictionaries, lists, and strings. In this way it is similar to JSON, YAML and TOML, but without the complexity and risk of YAML and without the syntactic clutter of JSON and TOML. NestedText is both simple and natural. Only a small number of concepts and rules must be kept in mind when creating it. It is easily created, modified, or viewed with a text editor and easily understood and used by both programmers and non-programmers.
```

(Obviously, adjust it on the 80-char limit.)

Reply:
Done. Thanks for the suggestion!